host-configs/TOTAL/pangea2-base.cmake

```diff
@@ -5,31 +5,31 @@
 #
 ########################################
 
-set( GEOSX_TPL_DIR /workrd/SCR/GEOSX/install/gcc8/GEOSX_TPL-d348871 CACHE PATH "")
```

Comment:
@sytuannguyen Are these all just whitespace changes?

Reply:
No he changed the MKL libraries. I think he just made the CMake syntax consistent with how we do it elsewhere.

Reply:
Yeah I added some space to make it consistent with the whole code and with itself because some space after ( and before ) are added, some are missing.

Reply:
It is lovely that GitHub choked on the non-whitespace diff.